ci-operator/config/redhat-chaos/lp-chaos/redhat-chaos-lp-chaos-main__ocp4.21-nightly--cnv-4.21-stable-windows-vm-chaos--aws.yaml

```
@@ -0,0 +1,64 @@
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+-releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.21"
+resources:
```

YAMLlint (1.37.1)

[error] duplication of key "resources" in mapping

(key-duplicates)